core/src/main/java/dk/alexandra/fresco/framework/network/CloseableNetwork.java

```diff
@@ -0,0 +1,16 @@
+package dk.alexandra.fresco.framework.network;
+
+import java.io.Closeable;
+
+/**
+ * Interface for a closable network.
+ */
+public interface CloseableNetwork extends Network, Closeable {
+
+  /*
+   * Override the close method to NOT throw IOExceptions (this may be kind of a hack)
```

Comment:
... hack?

Why circumvent it? If it is not closable, then do not inherit the interface

Reply:
It is closeable. I just dont want to have to throw an IOException.

Reply:
... I could just define my own interface and not inherit from Closeable, but I thought I would like to be able to use Classes implementing the interface in try-with-resources statements.

Reply:
Maybe use AutoCloseable, I mean the mere fact you are saying hack makes my nose cringe :)

Reply:
AutoCloaseable is not better. It also throws checked exception on close :(.

Reply:
But ok, it's not a huge deal to just handle the IOException. Maybe, the more interesting question is why `Network` does not extend `Closeable` already?